src/CalcViewModel/Common/NavCategory.cpp

```diff
@@ -72,17 +72,19 @@ bool IsGraphingModeEnabled()
         return _isGraphingModeEnabledCached->Value;
     }
 
-    User ^ firstUser;
-    create_task(User::FindAllAsync(UserType::LocalUser)).then([&firstUser](IVectorView<User ^> ^ users) {
-        firstUser = users->GetAt(0); }).wait();
-        auto namedPolicyData = NamedPolicy::GetPolicyFromPathForUser(firstUser, L"Education", L"AllowGraphingCalculator");
-        _isGraphingModeEnabledCached = namedPolicyData->GetBoolean() == true;
+    if (!firstUser)
+    {
+        return false;
```

Comment:
Since Graphing is available by default and is only disabled when specifically configured, does it make sense to set this to true if User is null?

Reply:
Resolved